OKR.md

```diff
@@ -0,0 +1,21 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q3
```

Comment:
Let's continue discussing the OKRs over the markdown, taking inspiration from the notes already shared in this thread.

I've defined 2 main objectives so far (rewording is welcome if you see the need) for this quarter which capture the most requested and needed features and bug fixes.

Let's shoot to get the final list of KRs by Wednesday, final review on Thursday and ship it on Friday.

Reply:
@diasdavid  should we add `IPNS parity and interoperability with go`?

Reply:
Absolutely @vasco-santos, good call!

I've updated the OKRs list and started assigning some based on previous work. Please review:

- Check if wording makes sense.
- Check if they are achievable this quarter.
- Check the ones you will want to take the lead.

Reply:
I believe that we can achieve this list!! 

I would like to take the lead on `DHT interops with go-libp2p`. 
By the way, is there any issue for the network visualization tool? I think I would like to go with it as well.

Reply:
Do you think it's worth having a 'technical debt' section?  There was something about speeding the tests up here a minute ago but it seems to have been removed.

Feature parity with the go API would also be a good thing to shoot for.

Reply:
@achingbrain you are confusing js-ipfs OKRs with js-libp2p OKRS. See https://github.com/ipfs/js-ipfs/pull/1409/files

Reply:
@vasco-santos awesome! Created an issue for you https://github.com/libp2p/js-libp2p/issues/215

Reply:
With this, I assigned the remaining to @jacobheun, hope you agree. I do think that we need to find more stuff to keep @jacobheun busy, otherwise he will be done in the first couple of weeks :D

Reply:
cool, thanks @diasdavid ! Then we sync about the tool 🙂

Reply:
> you are confusing js-ipfs OKRs with js-libp2p OKRS

Oops, yes.  I've added the comment there.

Reply:
@diasdavid haha, yes I'm good with taking the rest. I know NAT traversal was part of q2 but I'd like to make sure that gets done, as I think it's important to making libp2p the painless go to in p2p.

Reply:
rad!! And you are right, let's add NAT back in.